src/first_project.py

```diff
@@ -0,0 +1,43 @@
+"""Оформите решение задач из прошлых домашних работ в функции.
+
+Напишите функцию runner. (все проще когда мы изучим модули, getattr и setattr)
+
+runner() – все фукнции вызываются по очереди
+
+runner(‘func_name’) – вызывается только функцию func_name.
+
+runner(‘func’, ‘func1’...) - вызывает все переданные функции
+
+"""
+
+import tasks_module
+
+params = {
+    'swap_numbers': [1, 4, 0, 3, 6, 2, 0, 2],
+    'fibonacci': 15,
+    'remove_duplicate_characters': 'abc cde eft'
+}
+
+
+def runner(*args):
+    function = []
+    [function.append(elem) for elem in dir(tasks_module) if elem[:2] != '__']
+    try:
+        if args:
+            for elem in args:
+                callable(getattr(tasks_module, elem)(params.get(elem)))
```

Comment:
you first run and only then check it 
https://www.geeksforgeeks.org/callable-in-python/

Reply:
p.4 from previous review not addressed